packages/blocks/src/api/registration.js

```diff
@@ -174,10 +174,10 @@ export function unregisterBlockType( name ) {
 /**
  * Assigns name of block for handling non-block content.
  *
- * @param {string} name Block name.
```

Comment:
Why should we update this, but not `unregisterBlockType( name )` -> `unregisterBlockType( blockName )`. What's the motivation for this change?

Reply:
I will revert. It's obvious here what the name is. Other examples where I changed bring more clarity as the function name isn't as specific. e.g.: `setFreeformContentHandlerName( blockName )`.

Reply:
I didn't understand your question properly, but I think I answered it anyway :)

> It's obvious here what the `name` is.

`name` is also used in `registerBlockType` as the first param - so I wanted to have it consistent. In general I expanded to `blockName` in cases where the funcion name doesn't say anything about the block.